### PR TITLE
Rename redemption code to coupon ID

### PR DESCRIPTION
**Pull request by slavcodev**

### CODE REVIEW

File: openapi/components/schemas/Coupon/Coupon.yaml [hunk 1]
```diff
@@ -1,29 +1,29 @@
 type: object
-description: Coupons and Discounts
+description: Coupons and Discounts.
 required:
   - discount
   - issuedTime
 properties:
-  redemptionCode:
```
Comment (ashkarpetin) [inline openapi/components/schemas/Coupon/Coupon.yaml:7]: Shouldn't it be deprecated?

Comment (slavcodev) [reply openapi/components/schemas/Coupon/Coupon.yaml:7]: We do not keep deprecated properties in the API definition.

Comment (ashkarpetin) [reply openapi/components/schemas/Coupon/Coupon.yaml:7]: Do we?

https://github.com/Rebilly/api-definitions/blob/c8db940959f0153a7fd0e1b60ad57f19a1b38cb4/openapi/components/schemas/Common/CommonKycDocument.yaml#L16

https://github.com/Rebilly/api-definitions/blob/c8db940959f0153a7fd0e1b60ad57f19a1b38cb4/openapi/components/schemas/Common/CommonTransactionRequest.yaml#L36

Comment (slavcodev) [reply openapi/components/schemas/Coupon/Coupon.yaml:7]: This is by purpose, mistakenly or just very old definition. This is not very useful, we rather want to force people to migrate. The `deprecate` flag in OAS is useful only for code generation or when definiions are used in applications. 
Just for docs (our case) is useless and even make more harm than good.

Comment (slavcodev) [reply openapi/components/schemas/Coupon/Coupon.yaml:7]: I think this was discussed here in PR, should I find the PR to confirm?

**Review by ashkarpetin — APPROVED** [review]

**Review by colinbird — APPROVED** [review]

**Pull request merged**